docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+
+- Organization Admins
+- Organization Owners
+
+### Self-Managed Configuration
+
+Enable authorizations in your deployment using:
+
+```yaml
+camunda.security.authorizations.enabled: true
+```
+
+## Resources and Permissions
+
+### Available Resources
+
+The following table lists all resources that support authorization in Camunda 8 Orchestration Cluster, along with the available permissions per resource. This applies to Camunda 8 Orchestration Cluster (Zeebe, Operate, Tasklist, Orchestration Cluster APIs).
+
+| Resource Type                        | Resource Key Example                                    | Supported Permissions                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| **Application**                      | `Operate`, `Tasklist`, `Identity`, `*`                  | `ACCESS`                                                                                                                   |
+| **Authorization**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Batch**                            | `*`                                                     | `CREATE`, `READ`, `DELETE`                                                                                                 |
+| **Decision Definition**              | `*`, `decisionId: order_decision`                       | `CREATE_DECISION_INSTANCE`, `READ_DECISION_DEFINITION`, `READ_DECISION_INSTANCE`, `DELETE_DECISION_INSTANCE`               |
+| **Decision Requirements Definition** | `*`, `decisionRequirementsDefinitionId: order_decision` | `CREATE`, `READ`, `UPDATE`                                                                                                 |
+| **Group**                            | `*`, `groupId: accounting`                              | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Mapping Rules**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Message**                          | `*`                                                     | `CREATE`, `READ`                                                                                                           |
+| **Process Definition**               | `*`, `bpmnProcessId: order_process`                     | `CREATE_PROCESS_INSTANCE`, `READ_PROCESS_DEFINITION`, `READ_PROCESS_INSTANCE`, `READ_USER_TASK`, `UPDATE_PROCESS_INSTANCE` |
```

Comment:
🔧 I'm missing the `UPDATE_USER_TASK` permission

Reply:
resolved